app/assets/stylesheets/main.scss

```diff
@@ -120,10 +128,78 @@ body {
 
 .upvotes {
   text-align: center;
-  width: 15%;
-  padding: 10px 10px 1px 1px;
+  width: 30%;
+  padding: 15px 1px;
 }
 
 .no-deco {
   text-decoration: none;
+}
+
+.card {
+  border-radius: 0;
+}
+
+.comment-box {
+  display: inline-block;
+  background-color: $secondary-color;
+  color: $dark-color;
+  padding: 10px 60px 10px 15px;
+  margin: 10px 5px;
+  border-radius: 15px;
+}
+
+.chat-window {
+  border: 1px solid $secondary-color;
+  margin: 0 300px;
+  padding: 10px 0;
+  height: 50vh;
+  overflow-y: scroll;
+  max-width: 100%;
+  overflow-x: hidden;
+  display: flex;
+  flex-direction: column-reverse;
+}
+.border-window {
+  border: 1px solid $secondary-color;
+  padding: 10px 0;
+  margin: 0 300px;
+}
+
+.sent-message {
+  display: inline-block;
+  background-color: $base-color;
+  color: $light-color;
+  padding: 10px 20px;
+  margin: 20px 10px;
+  font-size: 1.2em;
+  border-radius: 15px;
+  text-align: left;
+}
+
+.sent-message-container {
+  text-align: right;
+}
+
+.received-message {
+  display: inline-block;
+  background-color: $secondary-color;
+  color: black;
+  padding: 10px 20px;
+  margin: 10px 20px;
+  font-size: 1.3em;
+  border-radius: 15px;
+  text-align: left;
+}
+
+.received-message-container {
+  text-align: left;
+}
+
+.fst-5 {
+  font-size: 0.8em;
+}
```

Comment:
font-size and class name doesnot match .fst-8

Reply:
this is just overriding default bootstrap class to my value